### ISSUE TITLE
Feature Request: max_norm argument added to tf.nn.nce_loss 

### ISSUE THREAD

**Issue by dshieble** [tensorflow/tensorflow#16805]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes

### Describe the problem
I would like to request an additional argument to added to the tf.nn.nce_loss function. This function calls the tf.nn.embedding_lookup in order to build the matrix of negative sample embeddings. tf.nn.embedding_lookup has a max_norm argument, and I would like for the value of the max_norm argument supplied to tf.nn.nce_loss to be propagated to the tf.nn.embedding_lookup call.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tatatodd** [comment]
Assigning to @xiejw, since he's touched some of this code, and may have a better idea than me.

**Comment by xiejw** [comment]
Re-assign to @ispirmustafa to find a proper owner.

**Comment by ispirmustafa** [comment]
Hi @dshieble,
Have you tried max norm with nce_loss in your application?
thanks

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ispirmustafa** [comment]
Closing due to inactivity.
